graf3d/csg/CMakeLists.txt

```diff
@@ -1,8 +1,8 @@
 ROOT_STANDARD_LIBRARY_PACKAGE(RCsg
   HEADERS
-     inc/CsgOps.h
+     CsgOps.h
   SOURCES
-     src/CsgOps.cxx
+     CsgOps.cxx
```

Comment:
What?! Dir-less `SOURCES` are used in many other modules, e.g. `graf2d/fitsio/CMakeLists.txt`. Can you explain?

Reply:
For headers we want to have them listed as included (to not hard-code paths), but for sources we want the full path to the file, otherwise we have to look for it with `find_file()`, which defeats the purpose of listing them. If other directories do not add the `src/` part, we should fix them to add it too.